### PR TITLE
Enhancement: Allow nesting of elements

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] allows nesting of elements

### CODE REVIEW

File: doc/guide/working_with_elements.rst [hunk 1]
```diff
@@ -135,6 +135,49 @@ Accessing custom elements is much like accessing inline ones:
...
+                'Logo' => '#logo',
+            ];
+
+            /**
+             * @return SearchForm
+             */
+            public function searchForm()
+            {
+                return $this->getElement(SearchForm::class);
+                // or (for PHP < 5.5.0) return $this->getElement('Page\\SearchForm')->search($keywords);
```
Comment (jakzal) [inline doc/guide/working_with_elements.rst]: It might be time to stop documenting PHP < 7.0

Comment (DonCallisto) [reply doc/guide/working_with_elements.rst]: We don't support it anymore so I strongly agree

Comment (localheinz) [reply doc/guide/working_with_elements.rst]: Removed!

**Comment by PWalkow** [comment]
What's the status of this one? Will it be merged?

**Comment by jakzal** [comment]
@PWalkow can you test it please?

**Comment by jakzal** [comment]
@PWalkow any luck? :)

**Comment by jakzal** [comment]
Thank you 🍻 

It will be included in the next release.

**Comment by DonCallisto** [comment]
Ops, just noticed it is. Sorry, my fault.
Cheers :beer:

**Comment by jakzal** [comment]
@DonCallisto it takes few minutes for that page to update :)

**Comment by localheinz** [comment]
Thank you, @DonCallisto, @jakzal, and @PWalkow!

**Pull request merged**